betel/categories_enum.py

```diff
@@ -0,0 +1,59 @@
+import enum
+
+
+class Categories(enum.Enum):
+    """Enum for the categories found in Google Play Store."""
```

Comment:
Explain how these categories are being used, i.e. the mapping is from enum to the string that we expect to be scraped

Reply:
Added to the docstring